README.md

```diff
@@ -1,13 +1,28 @@
 [![Build Status](https://travis-ci.com/bids-standard/bids-specification.svg?branch=master)](https://travis-ci.com/bids-standard/bids-specification)
 [![CircleCI](https://circleci.com/gh/bids-standard/bids-specification.svg?style=svg)](https://circleci.com/gh/bids-standard/bids-specification)
 
-![BIDS logo](BIDS_logo/BIDS_logo_black_transparent_background_crop.png)
 
-The Brain Imaging Data Structure (BIDS) is an emerging standard for the
+<img src="./BIDS_logo/BIDS_logo_black_transparent_background_crop.png" alt="bids-logo" width="600"/>
+
+The [Brain Imaging Data Structure (BIDS)](https://bids.neuroimaging.io) is an emerging standard for the
 organisation of neuroimaging data.
 
 In this repository, we develop the
 [BIDS specification](https://bids-specification.readthedocs.io/en/latest/).
 
-If you want to become a part of the BIDS community and contribute to the specification, please see our
-[contributing guidelines](./CONTRIBUTING.md).
+Want to learn more about working with BIDS? Have a question, comment, or suggestion?
```

Comment:
perhaps bold to bring attention to this? I think this is a typical case that should be highlighted

```suggestion
**Want to learn more about working with BIDS? Have a question, comment, or suggestion?**
```